### DISCUSSION TITLE
Can we add a new feature to dump the output to other location like kafka , s3 etc  ?

### DISCUSSION THREAD

**Discussion by dashbikash** [projectdiscovery/katana#951] [Ideas]
We need to add a new feature to dump the output to multiple destinations , which can be configurable.
The most popular options are described below in descending order.
1. KAFKA
2. MongoDB
3. HDFS
4. NATS
5. S3

**Comment by GeorginaReeder** [comment]
Hi @dashbikash - thanks for your suggestion.

Could you please go ahead and open a feature request for this? We'll be happy to take a closer look from there.

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
